src/socketworks.c

```diff
@@ -652,8 +655,11 @@ int sockets_del(int sock)
 	ss->is_enabled = 0;
 	so = ss->sock;
 	ss->sock = -1; // avoid infinite loop
-	LOG("sockets_del: sock %d -> handle %d, sid %d, overflow bytes %d, allocated %d, used %d, unsend packets %d",
-		sock, so, ss->sid, ss->overflow, ss->buf_alloc, ss->buf_used, (ss->wmax > 0) ? ((ss->wpos - ss->spos) % ss->wmax) : 0);
+	LOG("sockets_del: deleting socket sock %d (handle %d) [%s:%d], sid %d, overflow bytes %d, allocated %d, used %d, unsend packets %d",
```

Comment:
Changing wording makes grepping more difficult especially in the beginning. Suggesting to keep the same.

Reply:
That's is one of the more frustrating log lines. The current implementation is not clear. We need to now very clearly the ADDRESS:PORT of the connection, plus the full "Socket" and "Handle". So in this case the change of the wording it's necessary to improve the usability of the log.

Reply:
Adding more info is always helpful, but the wording is important for checking issues. I really doubt the users read those message and most likely will open an issue in the forum/github. that's why is better to make the job of finding issues a higher priority.